### PR TITLE
server-side database creating + instruction for API integration

### LINKED ISSUE #42: The "Create client" window remains opened after the client database had been successfully created
The "Create client" window remains opened after the client database had been successfully created (see img below).

image

I can just manually close the pop up window. But the thing is that if I click the "Create" button again by mistake, a 500 error will show on the screen.

### PR DESCRIPTION
**Pull request by yelizariev**
The update is not finished yet. Coming soon.

Also, it includes:
* fix #42 
* server-side database deleting

### CODE REVIEW

**Comment by yelizariev** [comment]
done. Reviews are welcome
@tarzan0820 @kaerdsar @dhbahr

File: README.md [hunk 1]
```diff
@@ -155,3 +155,82 @@ following modules on SaaS Client databases:
...
+                            [data, saas_portal_client_id])
+
+Notes abouts API integration
+
+* Be sure, that Portal module is installed at Main Database
+* Be sure, that "Allow external users to sign up" option from "Settings/General Settings" is enabled
+* To find new signuped user open "Settings/Users" at Main Database and delete filter "Regular users only"
+* don't use trailing slash at main_url
+* Access token is expired in one hour
+* In case of log out, client has to click "Log in via SaaS Portal". Client will be navigated to Portal database and can use client_username and client_password. After that the client will be returned back to his database. Important thing here, is that the client is not able to use client_password at login page of his database.
```
Comment (saltonmassally) [inline README.md:248]: The README is gradually becoming to large, I we should start using the Wiki

Comment (yelizariev) [inline README.md:248]: yeah, I thought about it too.
Wiki is not suit for this as such documentation is different for 8.0 and 9.0. So we probably need to use some tool to generate documentation.
In my opinion, readme is big, but not so much to start work on it right now

**Comment by yelizariev** [comment]
Key features have been tested, so I merge it.

**Pull request merged**